bakery/baking.py

```diff
@@ -23,17 +23,16 @@
     flatten,
     is_baked,
     is_cake,
+    is_piece_of_cake,
     replace_cakes,
 )
 
 
 def determine_baking_method(recipe: Any) -> BakingMethod:
     """Determine the first available baking method for recipe."""
-    if is_cake(recipe):
-        logger.info(
-            f"Cannot determine strategy for recipe {recipe} right now. "
-            "Bake recipe and try again later"
-        )
+    if is_cake(recipe) or is_piece_of_cake(recipe):
```

Comment:
Might be worth using any() and adding the list centrally, in case new functionality is needed

Reply:
moved to separate function